framework/CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ Yii Framework 2 Change Log
 - Enh #13702: Added support for PSR-3 'Logger' (klimov-paul)
 - Enh #13706: 'Profiler' layer extracted (klimov-paul)
 - Enh #15410: Added serialization abstraction layer under `yii\serialize\*` namespace (klimov-paul)
+- Chg #10659: Method `runAction()` now is invoked for every module instance in the request handling chain, allowing custom implementation in particular module (klimov-paul)
+- Enh #15438: Added middleware support including PSR-15 'HTTP Server Request Handlers' compatibility (klimov-paul)
```

Comment:
There are upcoming changes to be merged into PSR-15 spec soon.

Reply:
https://github.com/php-fig/fig-standards/pull/996

Reply:
The changes are only clarifications, and will not touch the actual interfaces and their relations to one another.